streamly-coreutils.cabal

```diff
@@ -0,0 +1,45 @@
+cabal-version:       2.4
+name:                streamly-coreutils
+version:             0.1.0.0
+synopsis:            GNU Coreutils like API for Streamly
+description:
+   Port useful commands from the GNU `coreutils` to Haskell functions using
+   streamly.
+
+   GNU `coreutils` are commonly used, well known interfaces to perform common
+   tasks on POSIX systems. It will be useful to have Haskell implementations of
+   these utilities in general and also to utilize the knowledge of these utilities
+   gained by programmers, admins, script writers for writing Haskell scripts.
+
+   Some of these, but not all, are available in some existing Haskell libraries in
+   some form or other. However, there is no comprehensive collection available,
+   performance may not be at par and implementations may not be streaming. The
+   goal of this project is to have a comprehensive streaming collection using
+   Streamly with performance competitive with that of the C implementations
+   where it matters.
+
+homepage:            https://github.com/composewell/streamly-coreutils
+bug-reports:         https://github.com/composewell/streamly-coreutils/issues
+license:             BSD-3-Clause
+license-file:        LICENSE
+author:              Composewell Technologies
+maintainer:          streamly@composewell.com
+copyright:           2020 Composewell Technologies
+category:            Control, Concurrency, Streaming, Reactivity
```

Comment:
We can remove control and reactivity from it.

Reply:
:+1: